message_app_client/src/routes/MessagesPage.js

```diff
@@ -33,13 +33,9 @@ function MessagesPage() {
 
   return (
     <div className='container'>
-      <ConversationDataContext.Provider value={{ conversationID, setConversationID }}>
+      <ConversationDataContext.Provider value={{ conversationID, setConversationID, otherParticipantName, setOtherParticipantName }}>
         <ConversationPanel {...{conversations: conversations}}/>
-        { conversationID ? (<ConversationDetailPanel
-          conversationId={1}
-          username={"user1"}
-          otherParticipantName={"user2"}
-        />) : <></> }
+        { conversationID ? (<ConversationDetailPanel />) : <></> }
```

Comment:
Could you display some form of notification here if there is no conversation selected?
Instead of returning `<></>`, we can return a more meaningful message.

Reply:
![image](https://user-images.githubusercontent.com/47405187/150884974-39f3b9c0-af2e-411e-beb6-0430e32c1e8e.png) 💯💯💯